markdown/3.x/en/documentation/server-side-rendering.md

```diff
@@ -35,7 +35,15 @@ Example libraries (the whole list of libraries can be found at [mustache.github.
 **Note**: You are not just limited to mustache. You can use any template engine your platform supports. The advantage of using mustache here is that you can use the same templates as in Web Components.  
 
 #### Sending the response
-The response to the browser contains the pre-rendered HTML.
+The response to the browser contains the pre-rendered HTML and raw response available to be used by Web Components.
+Most convenient way to make the response available to use by the Web Components is to store it as JSON in a variable.
+```html
+<script>
+const rawResponse = JSON.parse(RESPONSE_TEXT);
+</script> 
```

Comment:
Let's indent this to be consistent with other listings.
There is another listing that could be indented on line 50.
```suggestion
<script>
    const rawResponse = JSON.parse(RESPONSE_TEXT);
</script> 
```